agent-inject/agent/agent.go

```diff
@@ -271,6 +275,7 @@ func New(pod *corev1.Pod, patches []*jsonpatch.JsonPatchOperation) (*Agent, erro
 		LimitsCPU:              pod.Annotations[AnnotationAgentLimitsCPU],
 		LimitsMem:              pod.Annotations[AnnotationAgentLimitsMem],
 		Namespace:              pod.Annotations[AnnotationAgentRequestNamespace],
+    		Containers: 		[]string{pod.Annotations[AnnotationAgentInjectContainers]},
```

Comment:
I think this could be set directly here? Something like 

```go
Containers: = strings.Split(pod.Annotations[AnnotationAgentInjectContainers], ",")
```

That might make the line a little long here, so we could set it below instead. i.e.

```go
agent.Containers = strings.Split(pod.Annotations[AnnotationAgentInjectContainers], ",")
```